conf/study-defaults/consent-page.xhtml

```diff
@@ -0,0 +1,41 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <title>${studyName} Consent To Research</title>
+</head>
+<body>
+${consent.body}
```

Comment:
Again, the same question about "${consent.body}" being a regex.  Do we need to escape the dot?  Alternatively, could we use "${consentBody}" without the dot.